subgraphs/saucerswap/v2/src/utils/pricing.ts

```diff
@@ -59,9 +45,9 @@ export function sqrtPriceX96ToTokenPrices(sqrtPriceX96: BigInt, token0: Token, t
 
 export function getEthPriceInUSD(): BigDecimal {
   // fetch eth prices for each stablecoin
-  let usdcPool = Pool.load(USDC_WETH_03_POOL) // dai is token0
+  let usdcPool = Pool.load(USDC_WHBAR_03_POOL) // dai is token0
   if (usdcPool !== null) {
-    return usdcPool.token0Price
+    return usdcPool.token0Price //? maybe is token1?
```

Comment:
Q: were you able to confirm?

Reply:
Yes. I were able to confirm, is not, is token0 👍

Reply:
will remove the comment 👍

Reply:
✅ removed!